src/it/java/teammates/it/ui/webapi/GetSessionResponseStatsActionIT.java

```diff
@@ -0,0 +1,88 @@
+package teammates.it.ui.webapi;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.util.Const;
+import teammates.common.util.HibernateUtil;
+import teammates.storage.sqlentity.Course;
+import teammates.storage.sqlentity.Instructor;
+import teammates.storage.sqlentity.FeedbackSession;
+import teammates.ui.output.FeedbackSessionStatsData;
+import teammates.ui.webapi.GetSessionResponseStatsAction;
+import teammates.ui.webapi.JsonResult;
+
+
+/**
+ * SUT: {@link GetSessionResponseStatsAction}.
+ */
+public class GetSessionResponseStatsActionIT extends BaseActionIT<GetSessionResponseStatsAction> {
+
+    @Override
+    @BeforeMethod
+       protected void setUp() throws Exception {
+           super.setUp();
+           persistDataBundle(typicalBundle);
+           HibernateUtil.flushSession();
+       }
+    
+    @Override
+    String getActionUri() {
+        return Const.ResourceURIs.SESSION_STATS;
+    }
+
+    @Override
+    String getRequestMethod() {
+        return GET;
+    }
+
+    @Override
+    @Test
+    protected void testExecute() {
+        Instructor instructor = typicalBundle.instructors.get("instructor1OfCourse1");
+        loginAsInstructor(instructor.getGoogleId());
+
+        ______TS("typical: instructor accesses feedback stats of his/her course");
+
+        FeedbackSession accessibleFs = typicalBundle.feedbackSessions.get("session1InCourse1");
+        String[] submissionParams = new String[] {
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, accessibleFs.getName(),
+                Const.ParamsNames.COURSE_ID, accessibleFs.getCourse().getId(),
+        };
+
+        GetSessionResponseStatsAction a = getAction(submissionParams);
+        // TODO: current throws error at getGiverSetThatAnsweredFeedbackSession
```

Comment:
@kevin9foong As mentioned this `getGiverSetThatAnsweredFeedbackSession` throws null pointer exception, was wondering if you could spot any potential bugs in its current implementation🙏